content/docs/connectors/microsoft.md

```diff
@@ -124,3 +124,29 @@ connectors:
 
 Also, `useGroupsAsWhitelist` configuration option, can restrict the groups
 claims to include only the user's groups that are in the configured `groups`.
+
+If you need to match and deal with UPNs (User E-Mail) from the Active Directory, you
```

Comment:
There is a lot of information about Kubernetes. I think Kubernetes authentication is a common use-case for dex, but not the only one.

I would suggest you write documentation in the way:
1. Little description.
2. Why we need it.
3. Example.

The example of possible documentation (numbers are here just for clarity):
1. You can use the `emailToLowercase` (boolean) configuration option to streamline UPNs (user email) from Active Directory before putting them into an id token. 
2. Without this option, it can be tough to match the email claim because a client application doesn't know whether an email address has been added with capital- or lowercase letters.
3. For example, it is hard to bind Roles in Kubernetes using email as a user name (`--oidc-username-claim=email` flag) because user names are case sensitive.